app/views/events/show.html.erb

```diff
@@ -0,0 +1,64 @@
+<div class="centered">
+    <h1><%= @event.name %></h1>
+    <p>Description: <%= @event.description %></p>
+    <hr>
+    <div class="row space-between">
+        <p>Date: <%= @event.date %></p>
+        <% if @event.private? %>
+            <p>Type: Private</p>
+        <% else %>
+            <p>Type: Open</p>
+        <% end %>
+        <p>Creator:  <%= link_to @event.creator.name, @event.creator %> </p>
```

Comment:
To make this `@event.creator.name` call simpler we could have created a `@creator` instance variable in the controller (@creator = @event.creator).